### PR TITLE
Added preliminary Releasenotes for 10.9.1

### PR DESCRIPTION
**Pull request by tbsbdr**
Preliminary release notes for 10.9.1
**do not merge before release of 10.9.1**

Backport to 10.9 only

### CODE REVIEW

**Review by micbar — APPROVED** [review]
LGTM

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
Referencing: https://github.com/owncloud/docs/pull/4554 (Global Attribute Cleanup)

Before merging this PR, we should merge and backport the above one.
Than I can add a commit to change the complete bundle release name in site.yml pointing to the updated version.

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by jnweiger** [comment]
> Referencing: #4554 (Global Attribute Cleanup) - merged
> 
> I will add a commit to change the complete bundle release name in site.yml pointing to the updated version. (currently: `oc-complete-name: 'owncloud-complete-20211220'`)
> 
> To do so, I need the updated time stamp - please provide if available.

@mmattel final 10.9.1 will be built today. So the name  is  `owncloud-complete-20220112`

**Comment by mmattel** [comment]
The force push was necessary due to rebasing

File: modules/ROOT/pages/server_release_notes.adoc [hunk 1]
```diff
@@ -35,6 +36,25 @@ admin_manual:release_notes.adoc
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.9.1
+
+ownCloud Server 10.9.1 is a bug fix and maintenance release.
+You can read {owncloud-server-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Solved known issues
```
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:44]: There are some fixes to the new version-author feature that was released with 10.9.0.

Should that be mentioned somewhere in these release notes?

**Review by phil-davis — APPROVED** [review]
LGTM

**Pull request merged**